styx-scheduler-service/src/main/java/com/spotify/styx/monitoring/MetricsStats.java

```diff
@@ -122,8 +125,13 @@ public void registerQueuedEvents(Gauge<Long> queuedEventsCount) {
   }
 
   @Override
-  public void registerActiveStates(Gauge<Long> activeStatesCount) {
-    registry.register(ACTIVE_STATES, activeStatesCount);
+  public void registerActiveStates(RunState.State state, Gauge<Long> activeStatesCount) {
+    registry.register(ACTIVE_STATES_PER_RUNSTATE, activeStatesCount);
```

Comment:
This line will register different gauges for the same metricId. That will most likely lead to confusing results.